.github/workflows/build.yml

```diff
@@ -66,3 +64,43 @@ jobs:
           name: book-html
           path: |
             ./_build/html
+
+      - name: Deploy to gh pages
```

Comment:
@Revathyvenugopal162 this looks good. I think we want to use the new deploy to GitHub pages that we are using in our website repo [here](https://github.com/pyOpenSci/pyopensci.github.io/blob/main/.github/workflows/deploy-gh-pages.yml)

Are you open to modifying this to use deploy to GH pages rather than the peaceiris action instead?

Reply:
@lwasser ,sure, i can revert to that change.